### ISSUE TITLE
Add accessibility checking automation

### ISSUE THREAD

**Issue by Blackbaud-PatrickOFriel** [blackbaud/skyux1#528]
We might be able to add an accessibility rule check to our existing visual tests. Some tools to look at: 

http://www.deque.com/products/aXe/

https://github.com/springernature/pa11y

https://addyosmani.com/a11y/

These three tools can be installed through npm and run through the command line so they look to be promising in terms of easily integrating with our current build process.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.6.1](https://github.com/blackbaud/skyux/releases/tag/v1.6.1)
